lib/from_openstudio/model.rb

```diff
@@ -65,11 +66,12 @@ def self.translate_from_openstudio(openstudio_model)
       hash[:tolerance] = 0.01
       hash[:angle_tolerance] = 1.0
 
-      # Hashes for all shcedules and constructions in the model
+      # Hashes for various model properties
       $schedules = {}
       $opaque_constructions = {}
       $window_constructions = {}
       $shade_constructions = {}
+      $programtype_setpoint_hash = {}
```

Comment:
In OpenStudio, thermostats are not assignable to space types (there's always going to be a separate thermostat for each zone). So this seems like a variable that is not needed.